Assets/Events/Adventurers Join.json

```diff
@@ -190,7 +190,7 @@
 	{
 		"name": "bad-bard",
 		"headline": "New Bard In Town!",
-		"article": "Known around the lands for his experimental spoken word music, and stories about the stars. The reviews are in: <i>\"He's really bad.\"</i>",
+		"article": "The bard is known throughout the lands for his experimental spoken word music, and stories about the stars. Nobody is looking forward to their presence.",
```

Comment:
I don't know if this is funnier, I think the quoted bad review is a good joke but it requires a little something else

Reply:
The blunt end comes off as a bit childish to me. To me, the funny part is that they're initially framed as some kind of cool, experimental artist, and then cut it with revealing that it's viewed as awful and/or pretentious.

Reply:
I have however noticed a gender disagreement I didn't fix that I'll do now.